templates/admin/org/list.tmpl

```diff
@@ -52,7 +52,7 @@
 						<tr>
 							<td>{{.ID}}</td>
 							<td>
-								<a href="{{.HomeLink}}">{{.Name}}</a>
+								<a href="{{.HomeLink}}">{{if and DefaultShowFullName .FullName}}{{.FullName}}({{.Name}}){{else}}{{.Name}}{{end}}</a>
```

Comment:
Space is missing

Reply:
Which space did you mean?

Reply:
Do you write sentences like "something (something else)" or "something(something else)"?

Reply:
d8c9044637